src/content/docs/style-guide/images/create-videos.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Create videos
+tags:
+  - Tech writer style guide
+  - Processes and procedures
+redirects:
+---
+
+import styleGuideCamtasiaEditing from 'images/style-guide_screenshot-crop_camtasia-editing.png'
+
+import styleGuideCamtasiaBlur from 'images/style-guide_screenshot-full_camtasia-blur.png'
+
+
+There are many ways to create videos, but here are some tips that will likely apply to a lot of video production projects. 
```

Comment:
@zuluecho9 The word "ways" is pretty general and could encompass tools. How about making this focused on best practices: To help you avoid feeling overwhelmed by all the best practices for video production, here are some tips that will likely apply to a lot of video production projects.